dnscrypt-proxy/example-blocked-ips.txt

```diff
@@ -1,8 +1,8 @@
 ##############################
-#        IP blacklist        #
+#        IP blocklist        #
 ##############################
 
-## Response containing blacklisted IP addresses will be blocked
+## Response containing blocklisted IP addresses will be blocked
```

Comment:
Something like rules for IP-based response blocking maybe? (Obviously "blocked IP addresses will be blocked" wouldn't work!)

Reply:
agreed - "Rules for IP-based response blocking"